docker-compose.hobby.yml

```diff
@@ -145,6 +145,7 @@ services:
         environment:
             SENTRY_DSN: $SENTRY_DSN
             SITE_URL: https://$DOMAIN
+            TEMPORAL_HOST: temporal
```

Comment:
Reading through the issue, we may need `SECRET_KEY: $POSTHOG_SECRET` here too. Have you been able to verify this @jk2K? If not, I can run a test of the hobby stack later today and decide.

Reply:
got error. 
```
ValueError: Duplicated timeseries in CollectorRegistry: {'posthog_celery_task_pre_run_created', 'posthog_celery_task_pre_run_total', 'posthog_celery_task_pre_run'}
```